packages/core/src/browser/browser-clipboard-service.ts

```diff
@@ -84,13 +84,12 @@ export class BrowserClipboardService implements ClipboardService {
                     It can be enabled by 'dom.events.testing.asyncClipboard' preference on 'about:config' page. Then reload Theia.
                     Note, it will allow FireFox getting full access to the system clipboard.`);
                 }
-                throw new Error('Failed writing the the clipboard.');
+                throw new Error('Failed writing the clipboard.');
```

Comment:
`to`?

Reply:
I wonder for `isFirefox` case should not we return to avoid throwing an exception to clients? The same question for read text.

Reply:
Agree. Fixed.